app/components/community_activity_component/community_activity_component.html.erb

```diff
@@ -122,17 +123,28 @@
                               <% end %>
                             </ul>
                           <% end %>
-                          <textarea class="govuk-textarea" placeholder="Please provide us with evidence of the activity you delivered." data-community-activity-component-target="textarea"><%= @achievement&.evidence&.[](index) %></textarea>
                         </p>
+                        <textarea class="govuk-textarea govuk-!-margin-bottom-2" placeholder="Please provide us with evidence of the activity you delivered." data-community-activity-component-target="textarea"><%= @achievement&.evidence&.[](index) %></textarea>
+                        <div class="community-activity-component__completion-count govuk-!-margin-bottom-1">
+                          <span class="govuk-body-s" data-community-activity-component-target="characterCountMessage"></span>
+                        </div>
+                      <% end %>
+                    <% end %>
+
+                    <% if minimum_character_requirement != 0 %>
```

Comment:
Is this if statement needed, as we don't show the warning when its not needed? Might be in the future we want to use the warning step for other information that do not relate to the min char requirement?